includes/functions/core.php

```diff
@@ -44,9 +44,9 @@ function init() {
 }
 
 function i18n() {
-	$locale = apply_filters( 'plugin_locale', get_locale(), 'eight-day-week' );
-	load_textdomain( 'eight-day-week', WP_LANG_DIR . '/print-production/print-production-' . $locale . '.mo' );
-	load_plugin_textdomain( 'eight-day-week', false, plugin_basename( EDW_PATH ) . '/languages/' );
+	$locale = apply_filters( 'plugin_locale', get_locale(), 'eight-day-week-print-workflow' );
+	load_textdomain( 'eight-day-week-print-workflow', WP_LANG_DIR . '/eight-day-week-print-workflow/eight-day-week-print-workflow-' . $locale . '.mo' );
```

Comment:
I'm least confident in this change here, so confirmation that the concatenation of this all makes sense would be good.

Reply:
I think this is right, it works correctly locally. I can check some other plugins as well to confirm the approach.

Reply:
I don't think we need this anymore if trying to use WordPress.org language packs? It means bumping the WP required version (4.6+ I believe) and then we can remove the text domain header. I'm going to ask somebody who knows more about how the .org side about why this doesn't work with the custom text domain header set.

Reply:
Sounds good, not super urgent to get these changes handled, just more a "nice to check it off the list" sort of thing.